src/webgpu/api/validation/image_copy/image_copy.ts

```diff
@@ -156,7 +156,7 @@ export function texelBlockAlignmentTestExpanderForValueToCoordinate({
 
 // This is a helper function used for filtering test parameters
 export function formatCopyableWithMethod({ format, method }: WithFormatAndMethod): boolean {
-  if (method === 'CopyTextureToBuffer') {
```

Comment:
The original code was true because copySrc and copyDst are always the same for now.